build/Makefile.kits

```diff
@@ -40,6 +41,12 @@ else
   OUTPUT_PACKAGE_PREFIX=$(SCX_PREFIX)-$(SCX_BUILDVERSION_MAJOR).$(SCX_BUILDVERSION_MINOR).$(SCX_BUILDVERSION_PATCH)-$(SCX_BUILDVERSION_BUILDNR).universal.$(PF_ARCH)
 endif
 
+# InstallBuilder has it's own concept of naming of installer files, but
+# we will override that in certain cases via the --OUTPUTFILE option
+ifneq ($(OUTPUT_PACKAGE_PREFIX),"")
```

Comment:
Does InstallBuilder fail when `OUTPUT_PACKAGE_PREFIX == ""`?
Fixing this in Install builder seems more appropriate.